x/leverage/keeper/collateral.go

```diff
@@ -144,9 +144,9 @@ func (k Keeper) CollateralLiquidity(ctx sdk.Context, denom string) sdk.Dec {
 	exchangeRate := k.DeriveExchangeRate(ctx, denom)
 	liquidity := k.AvailableLiquidity(ctx, denom)
 
-	// Zero collateral will be interpreted as having no liquidity
+	// Zero collateral will be interpreted as full liquidity
```

Comment:
please add more details why? This doesn't look logical at first glance.

Reply:
I'll add the comment - but basically, if you have 100 liquidity and 0 collateral, then "all" collateral is liquid. (The scenario overall is healthy). This can also apply to 0 collateral and 0 liquidity.

`0/0` shouldn't cause `ErrMinCollateralLiquidity`, which is what I mistakenly had it doing.